controllers/snapshot/snapshot_adapter.go

```diff
@@ -507,6 +508,11 @@ func (a *Adapter) createMissingReleasesForReleasePlans(application *applicationa
 			}
 			a.logger.Info("Marked Release status automated", "release.Name", newRelease.Name)
 		}
+		//get time tests finished and use as start time to calculate release created latency
+		startTime, ok := gitops.GetAppStudioTestsFinishedTime(a.snapshot)
```

Comment:
Does it make more sense to title this function `GetReleaseStartTime?`

Reply:
No, function returns time when test finished. It has nothing with release